src/libasr/codegen/asr_to_c_cpp.h

```diff
@@ -116,6 +116,7 @@ class BaseCCPPVisitor : public ASR::BaseVisitor<Struct>
     std::string const_name;
     size_t const_vars_count;
     size_t loop_end_count;
+    int bracket_open;
```

Comment:
Can you please add documentation here what this variable is used for, where it needs to be incremented/decremented (essentially what is the "contract" that a developer has to follow when adding new functionality)?

Reply:
Sure, will do that. Thanks